src/openrct2/ride/CarEntry.h

```diff
@@ -144,20 +148,22 @@ enum class SpriteGroupType : uint8_t
     Slopes60Banked22,
     Corkscrews,
     RestraintAnimation,
-    CurvedLiftHill,
+    CurvedLiftHillUp,
+    CurvedLiftHillDown,
     Count
 };
 
 static const std::string SpriteGroupNames[] = {
-    "slopeFlat",        "slopes12",         "slopes25",           "slopes42",
-    "slopes60",         "slopes75",         "slopes90",           "slopesLoop",
-    "slopeInverted",    "slopes8",          "slopes16",           "slopes50",
-    "flatBanked22",     "flatBanked45",     "flatBanked67",       "flatBanked90",
-    "inlineTwists",     "slopes12Banked22", "slopes8Banked22",    "slopes25Banked22",
-    "slopes8Banked45",  "slopes16Banked22", "slopes16Banked45",   "slopes25Banked45",
-    "slopes12Banked45", "slopes25Banked67", "slopes25Banked90",   "slopes25InlineTwists",
-    "slopes42Banked22", "slopes42Banked45", "slopes42Banked67",   "slopes42Banked90",
-    "slopes60Banked22", "corkscrews",       "restraintAnimation", "curvedLiftHill",
+    "slopeFlat",          "slopes12",         "slopes25",           "slopes42",
+    "slopes60",           "slopes75",         "slopes90",           "slopesLoop",
+    "slopeInverted",      "slopes8",          "slopes16",           "slopes50",
+    "flatBanked22",       "flatBanked45",     "flatBanked67",       "flatBanked90",
+    "inlineTwists",       "slopes12Banked22", "slopes8Banked22",    "slopes25Banked22",
+    "slopes8Banked45",    "slopes16Banked22", "slopes16Banked45",   "slopes25Banked45",
+    "slopes12Banked45",   "slopes25Banked67", "slopes25Banked90",   "slopes25InlineTwists",
+    "slopes42Banked22",   "slopes42Banked45", "slopes42Banked67",   "slopes42Banked90",
+    "slopes60Banked22",   "corkscrews",       "restraintAnimation", "curvedLiftHillUp",
+    "curvedLiftHillDown",
```

Comment:
Doesn't this change mean any existing objects that have a curved lift hill sprite group will import incorrectly. Also this is touching the scripting api.

Reply:
Any vehicle that uses the _new_ sprite groups and has curved lift hills will import incorrectly. I believe there are none that exist. Legacy OpenRCT2 groups and .DAT are unaffected.

Reply:
@X123M3-256 I assume you are the only other person that would know about and use the sprite groups. Could you confirm you haven't used curvedLiftHill in any of your parkobj objects?

Reply:
@X123M3-256 has confirmed he hasn't used it.